src/app/components/feed/jh-feed.directive.js

```diff
@@ -61,6 +61,10 @@
             // Reset the warning timeout
             mutedWarningTimeout = now();
           });
+          scope.$on('muted.Join', function() {
+            mutedWarningTimeout = secondsFromNow(60);
```

Comment:
The reasoning behind this was to stop a second notification (when voice is detected) from appearing while the first is not dismissed yet. But now I see that it didn't fix it properly so I will take another approach..